modules/nw-sriov-topology-manager.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple-networks/configuring-sr-iov.adoc
+
+[id="nw-sriov-topology-manager_{context}"]
+= Creating NUMA aligned SR-IOV pod
+
+You can create a NUMA aligned SR-IOV pod by restricting SR-IOV and the CPU resources allocated from the same NUMA node with `restricted` or `single-numa-node` Topology Manager polices.
```

Comment:
What's NUMA mean?

Reply:
I'll confirm with the team, but I think it's non-uniform memory access. And I'll spell it out here either way.